run_service.sh

```diff
@@ -604,6 +604,9 @@ export CUSTOM_GNOSIS_SAFE_SAME_ADDRESS_MULTISIG_ADDRESS="0x6e7f594f680f7aBad18b7
 export CUSTOM_MULTISEND_ADDRESS="0x40A2aCCbd92BCA938b02010E17A5b8929b49130D"
 export MECH_CONTRACT_ADDRESS="0x77af31De935740567Cf4fF1986D04B2c964A786a"
 export WXDAI_ADDRESS="0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d"
+export DISABLE_TRADING=false
+export STOP_TRADING_IF_STAKING_KPI_MET=true
+export RESET_PAUSE_DURATION=300
```

Comment:
Why do you want to override this?

Reply:
This ensures that the trader waits 5 minutes before restarting the cycle, to avoid unnecessary cycles, excessive logging, etc.

Reply:
That would also slow down the trades though.

Reply:
Does this only affect the agents in staked services? That would be acceptable, and it can always be changed.

If it also affects the non-staked agents, then this would be unnecessary long IMO

Reply:
It affects all agents. The rational is that waits for some time before trying to go through the trading cycle. Feel free to modify with a lower value / remove it, if you find it appropriate.